en/debug/gdb_debugging.md

```diff
@@ -24,52 +18,24 @@ top
 
 Stack usage is calculated with stack coloring and thus is not the current usage, but the maximum since the start of the task.
 
-### Heap allocations
```

Comment:
YOu moved this section into Simulation debugging, which is great. Does this topic need to be covered for the hardware case too - or is that done by the console commands you added?

Reply:
I would not know how it can be done on hardware.